packages/components/dropdown/src/dropdown-item.vue

```diff
@@ -5,7 +5,7 @@
   >
     <el-roving-focus-item :focusable="!disabled">
       <el-dropdown-item-impl
-        v-bind="$props"
+        v-bind="$attrs"
```

Comment:
I thin we need to do:

```html
v-bind="{...$props, ...$attrs}"
```

Allowing the `$props` to fallthrough.

Reply:
@mawi1512 Would you mind change the code as I commented? I think your code are just fine.

Reply:
@JeremyWuuuuu You are right, $props was missing. I tried to use v-bind="{...$props, ...$attrs}", but it caused type errors. I hope the alternative solution is also ok. Let me know what you think.

Reply:
Looking good, test has been triggered, if everything was fine then we should be able to merge this.